src/Components/Notifications/util.ts

```diff
@@ -14,3 +17,28 @@ const isArtworksBasedNotification = (
 ) => {
   return ["ARTWORK_ALERT", "ARTWORK_PUBLISHED"].includes(notificationType)
 }
+
+export const setLastSeenNotificationDate = date => {
+  if (!date) {
+    return
+  }
+
+  window.localStorage.setItem(LAST_SEEN_NOTIFICATION_PUBLISHED_AT_KEY, date)
+}
+
+export const hasNewNotifications = lastNotificationDatetime => {
```

Comment:
This whole function could just be a compound boolean statement rather than a series of conditionals that return booleans.

Reply:
There should be no `any` types in this codebase.